include/c2pa.hpp

```diff
@@ -277,27 +277,34 @@ namespace c2pa
         /// @brief  Set the no embed flag.
         void set_no_embed();
 
-        /// @brief  Set the remote URL.
+        /// @brief Set the remote URL.
         /// @param remote_url  The remote URL to set.
         /// @throws C2pa::C2paException for errors encountered by the C2PA library.
         void set_remote_url(const string &remote_url);
 
-        /// @brief  Add a resource to the builder.
-        /// @param uri  The uri of the resource.
-        /// @param source  The input stream to read the resource from.
+        /// @brief Set the base path for loading resources.
```

Comment:
...for loading resources from files.
The default is to load them only from memory if this is not set.

Reply:
Done.